src/PhpWord/Shared/Html.php

```diff
@@ -869,7 +870,11 @@ protected static function parseImage($node, $element)
         }
 
         if (is_file($src)) {
-            $newElement = $element->addImage($src, $style);
+            try {
+                $newElement = $element->addImage($src, $style);
+            } catch (InvalidImageException $ex) {
+                $newElement = $element->addText($src);
```

Comment:
I guess just returning the string src is the bare minimum, perhaps we could prefix it like 
```suggestion
                $newElement = $element->addText("Error: Could not load image: {$src}");
```

Reply:
Good shout, will do